Emittery.d.ts

```diff
@@ -76,6 +76,16 @@ declare class Emittery {
 	 * specified.
 	 */
 	listenerCount(eventName?: string): number;
+
+	/**
+	 * Bind a `methodNames` or all `Emittery` methods if `methodNames` not defined into `target` object.
+	 */
+	bindMethods(target: Object, methodNames?: string[]): void;
+
+	/**
+	 * Returns a decorator which mixins `Emittery` as property `emitteryPropertyName` and `methodNames` or all `Emittery` methods if `methodNames` not defined into target class.
+	 */
+	static mixin(emitteryPropertyName: string, methodNames?: string[]): Function;
```

Comment:
Can you move this to the top? I prefer having static properties first.
```suggestion
	static mixin(emitteryPropertyName: string, methodNames?: readonly string[]): Function;
```

Reply:
Is there any way to make the type more specific than `Function`?

Reply:
It is not supposed to be more specific than function, I guess. There shouldn't be any type checks on decorators.

Reply:
You're right. Probably not worth it.